thirdparty/scripts/build_credis.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -x
+
+# Cause the script to exit if a single command fails.
+set -e
+
+TP_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)/../
+ROOT_DIR=$TP_DIR/..
+
+if [[ "${RAY_USE_NEW_GCS}" = "on" ]]; then
+  pushd "$TP_DIR"
+    rm -rf credis
```

Comment:
Wouldn't it be better to check if credis is already present and if so then just skip the build? That would speed up our builds.. although we don't build credis in the common case so either way is fine.

Reply:
fixed